BPOR/BPOR.Registration.Stream.Handler/src/BPOR.Registration.Stream.Handler/Handlers/StreamHandler.cs

```diff
@@ -146,6 +152,8 @@ private async Task ProcessRemoveAsync(DynamoDBEvent.DynamodbStreamRecord record,
         var participant = await participantDbContext.GetParticipantByLinkedIdentifiers(identifiers)
             .IgnoreQueryFilters()
             .Include(x => x.ParticipantIdentifiers)
+            .Include(x => x.ContactMethods)
+            .Include(x => x.ParticipantLocation)
```

Comment:
Are we doing anything with location in this method? If not we don't need to include it.

Reply:
removed location include